test/multiverse/suites/ruby_openai/ruby_openai_instrumentation_test.rb

```diff
@@ -233,4 +233,33 @@ def test_set_llm_agent_attribute_on_embedding_transaction
 
     assert_truthy harvest_transaction_events![1][0][2][:llm]
   end
+
+  def test_embeddings_drop_input_when_record_content_disabled
+    with_config(:'ai_monitoring.record_content.enabled' => false) do
```

Comment:
It would be good to have the 2 `enabled => true` inverse tests here as well. That way we don't rely on the defaults and sometimes even catch things that an all-refute approach might miss.

Reply:
I like this idea! Ill add the inverse tests